content/SCALE/ScaleSecurityReports/_index.md

```diff
@@ -4,6 +4,6 @@ geekdocCollapseSection: true
 weight: 135
 ---
 
-SCALE is currently in BETA and only recommended for early adopters who have a backup plan.
+TrueNAS SCALE is not currently an enterprise release. We recommended that SCALE is used by early adopters who have a backup plan.
```

Comment:
I recommend:

Changing

`We recommended that SCALE is used by early adopters who have a backup plan.`

To

`We only recommended SCALE for early adopters who have a backup plan.
`